README.md

```diff
@@ -15,6 +15,17 @@ super-fast querying and searching, as well as serves the merkle proofs needed to
 The API specification is located here https://github.com/metaplex-foundation/api-specifications
 This spec is what providers of this api must implement against.
 
+#### GRPC-INGEST [/grpc-ingest/README.md](/grpc-ingest/README.md)
+
+This repo also houses the GRPC-INGEST component. This is a reimplementation of nft-ingester
+This components separates grpc and ingester into two separate components. This is to allow for more flexibility in the future also increases the performance of the system.
+
+The two components are:
+
+- grpc2redis - This component listens to a gRPC stream (triton's Dragon's Mouth gRPC) for account and transaction updates and pushes them to a pipeline that flushes data to redis at regular intervals in real time.
+
+- ingester - This component listens to the redis stream and processes the data using program transformers. It also downloads token metadata json and stores them in a postgres db using sea-orm.
```

Comment:
Please remove from the main read me but make sure the contents are covered in the grpc-ingeest README.